org.scala-ide.sdt.core/src/org/scalaide/core/compiler/NamePrinter.scala

```diff
@@ -191,6 +218,7 @@ class NamePrinter(cu: InteractiveCompilationUnit) {
         case comp.Import(tree, selectors) => handleImport(loc, tree, selectors)
         case ident: comp.Ident => handleIdent(ident)
         case packageDef: comp.PackageDef => handlePackageDef(packageDef)
+        case apply: comp.Apply => handleApply(apply)
```

Comment:
You might add here `TypeApply`, or maybe a supertype of the two, like `GenericApply`